hathor/builder/cli_builder.py

```diff
@@ -163,6 +164,19 @@ def create_manager(self, reactor: PosixReactorBase, args: Namespace) -> HathorMa
         soft_voided_tx_ids = set(settings.SOFT_VOIDED_TX_IDS)
         consensus_algorithm = ConsensusAlgorithm(soft_voided_tx_ids, pubsub=pubsub)
 
+        if args.x_enable_event_queue:
+            if not settings.ENABLE_EVENT_QUEUE_FEATURE:
+                self.log.error('The event queue feature is not available yet')
+                sys.exit(-1)
+
+            self.log.info('--x-enable-event-queue flag provided. '
+                          'The events detected by the full node will be stored and can be retrieved by clients')
```

Comment:
Should we say that "previous events were cleared out" when `args.x_reset_event_queue is True`?

Reply:
This CLI arg was removed in https://github.com/HathorNetwork/hathor-core/pull/608/commits/79f2773a773adcbef4f5d853344e51bb66cfb4d2

Reply:
Ok. Thanks!